Zend/zend_builtin_functions.stub.php

```diff
@@ -7,6 +7,11 @@ class stdClass
 {
 }
 
+function exit(string|int $code = 0): never {}
+
+/** @alias exit */
+function die(string|int $code = 0): never {}
```

Comment:
The parameter names seem to be incorrect.

It should be `$status`, not `$code`.

The RFC explicitly uses that parameter name in the proposal: https://wiki.php.net/rfc/exit-as-function#proposal

It is also the name already used in the [manual](https://www.php.net/exit).

Lastly, the parameter name `$status` better covers what can be passed: either a status _message_ or a status _code_.
While `$code` would read pretty weird when passing a message:
```php
exit(code: 'message');
```

Reply:
@Girgias Note: I'll happily try to provide a PR to fix this, including adding a test with named parameter use, but I wanted to check first if this was an oversight or if there was a particular reason for the parameter name change.

Reply:
https://github.com/php/php-src/pull/15433 (draft as I've guestimated a patch ;-) )